tests/output/shared_elastic.py

```diff
@@ -39,7 +39,7 @@ class SharedElasticsearchOutputModuleTest(test_lib.OutputModuleTestCase):
       {'data_type': 'syslog:line',
        'display_name': 'log/syslog.1',
        'filename': 'log/syslog.1',
-       'hostname': 'ubuntu',
+       'hostname': b'ubuntu',
```

Comment:
please add an separate attribute "binary_data" and add a comment what it is used for